src/index.js

```diff
@@ -85,7 +86,10 @@ async function startApp() {
         defaultTimezone = timezone
       }
 
-      const loaders = createLoaders(accessToken, userID, requestID)
+      const loaders = createLoaders(accessToken, userID, {
```

Comment:
This was confusing, here its `requestID`, and below its `requestIDs`.

However, they wind up going thru the same factory (where it's `requestIDs`), not sure what the deal is?

Either way, testing some stuff locally, I was seeing request id and user agent in the outgoing requests 👍

Reply:
🤔 weird.

Reply:
This doesn't look like it was actually used except in convection/merged schemas, so it's possible that its wrong (and should be `requestIDs`, or whatever), or maybe this is actually right! And it's being used properly in the `mergeSchemas` methods.